Makefile

```diff
@@ -196,11 +196,13 @@ godocs:
 build-docs:
 	@cd docs && \
 	while read -r branch path_prefix; do \
-		(git checkout $${branch} && npm install && VUEPRESS_BASE="/$${path_prefix}/" npm run build) ; \
+		echo "building branch $${branch}" ; \
+		(git clean -fdx && git reset --hard && git checkout $${branch} && npm install && VUEPRESS_BASE="/$${path_prefix}/" npm run build) ; \
```

Comment:
this prevents docs build from breaking if there are uncommitted git diffs from building docs (currently occurring)

It will also reset your unsaved local changes if you run this. Copied from SDK